LICENCE

```diff
@@ -0,0 +1,8 @@
+/*******************************************************
+ * Copyright (C) 2017-2018 James Thorogood <jamesthorogood@ewb.ca>
```

Comment:
Shouldn't these licenses be for Boresha instead of JT directly?

Reply:
Sorry, for some reason I didn't get notifications for these comments. The only reason I put JT's name instead of Boresha is because Boresha technically isn't an incorporated company (I think). I may be wrong about this. If I am, I will change the licence accordingly.

Reply:
ah okay, make sure you double check with JT soon! Perhaps maybe even before merging (if not, make sure you make an issue for it so we can keep track of it)